src/utils.cairo

```diff
@@ -237,3 +237,20 @@ pub fn from_base16(hexs: ByteArray) -> @ByteArray {
 
     @result
 }
+
+pub fn from_string_to_hexa(hexs: ByteArray) -> @ByteArray {
```

Comment:
name should be `from_hex_to_byte` or something like that no?
current name suggests we convert a string to an hex